test/JDBC/expected/BABEL-5186-vu-verify.out

```diff
@@ -0,0 +1,2460 @@
+-- Error: relation \"%s\" does not exist
```

Comment:
have we validated same tests against SQL server?
Please add following variations as well
batch -> proc (with try/catch) -> proc (without try catch and throws error)
proc (without try/catch) -> proc (with try/catch) -> proc (without try catch and throws error)
- add few tests with functions having try/catch
- add few tests with triggers having try/catch
In catch part try following commands (with and without active transaction)
- rollback, commit, DMLs, queries (table access), simple queries (no table access) (have both variations where these commands are added at nest level 1 or nest level 2 (assuming we have three level nesting)
- Add few tests with savepoints
- Add test for similar unmapped error which should behave the old way

Reply:
Added above mentioned tests.